lib/flux/app-state.js

```diff
@@ -696,6 +696,78 @@ export const actionMap = new ActionMap({
         tags: { $set: tags },
       });
     },
+
+    loadPreferences: {
+      creator({ callback, preferencesBucket }) {
+        return dispatch => {
+          const objectKey = 'preferences-key';
+
+          preferencesBucket.get(objectKey, (e, preferences) => {
+            const preferencesData = preferences ? preferences.data : {};
+            const {
+              analytics_enabled: analyticsEnabled = true,
```

Comment:
Unfortunately we need to check if this is either a boolean or a int value of `0` or `1` because of how iOS saves the boolean value (it won't be fixed in the near future). We can still save it as a boolean because iOS can read the value properly still.

Reply:
Added a step to cast it to a boolean 👍

Reply:
Javascript can be so magical sometimes ❤️